documentation/asciidoc/accessories/camera/synchronous_cameras.adoc

```diff
@@ -1,102 +1,97 @@
 
-Both the HQ Camera and the Global Shutter Camera, have support for synchronous captures.
-Making use of the XVS pin (Vertical Sync) allows one camera to pulse when a frame capture is initiated.
-The other camera can then listen for this sync pulse, and capture a frame at the same time as the other camera.
+The High Quality (HQ) Camera supports synchronous captures.
+Using the XVS pin (Vertical Sync) allows one camera (the "source") to pulse when a frame capture is initiated.
+Other ("sink") cameras can listen for this pulse, and capture a frame at the same time as the source camera.
 
-=== Using the HQ Camera
+[NOTE]
+====
+Global Shutter (GS) Cameras can also be operated in a synchronous mode. However, the source camera will record one extra frame. Instead, for the GS Camera we recommend using an xref:camera.adoc#external-trigger-on-the-gs-camera[external trigger source]. You cannot synchronise a GS Camera to an HQ Camera.
+====
 
-For correct operation, both cameras require a 1.65V pull up voltage on the XVS line, which is created by a potential divider through the 3.3V and GND pins on the Raspberry Pi.
+=== Connecting the cameras
 
-image::images/synchronous_camera_wiring.jpg[alt="Image showing potential divider setup",width="50%"]
+Solder a wire to the XVS test point of each camera, and connect them together.
 
-Create a potential divider from two 10kΩ resistors to 3.3V and ground (to make 1.65V with an effective source impedance of 5kΩ). This can be connected to either Raspberry Pi.
+Solder a wire to the GND test point of each camera, and connect them together.
 
-Solder the GND and XVS test points of each HQ Camera board to each other.
+*For GS Cameras only,* you will also need to connect the XHS (Horizontal Sync) test point of each camera. On any GS Camera that you wish to act as a sink, bridge the two halves of the MAS pad with solder.
 
-Connect the XVS wires to the 1.65V potential divider pull-up.
+NOTE: An earlier version of this document recommended an external pull-up for XVS. This is no longer recommended. Instead, ensure you have the latest version of Raspberry Pi OS and set the `always-on` property for all connected cameras.
 
-==== Boot up both Raspberry Pis
+=== Driver configuration
 
-The file `/sys/module/imx477/parameters/trigger_mode` determines which board outputs pulses, or waits to receive pulses (source and sink).
-This parameter can only be altered in superuser mode.
+You will need to configure the camera drivers to keep their 1.8V power supplies on when not streaming, and optionally to select the source and sink roles.
 
-Run the following commands to configure the sink:
+==== For the HQ Camera
 
-[source,console]
+As the superuser, edit `/boot/firmware/config.txt`. Change `camera_auto_detect=1` to `camera_auto_detect=0`.
+
+Append this line for a source camera:
+[source]
 ----
-$ sudo su
-$ echo 2 > /sys/module/imx477/parameters/trigger_mode
-$ exit
+dtoverlay=imx477,always-on,trigger-mode=1
```

Comment:
Whoops. I've just spotted this is an error. Although the OF property and module parameter are called `trigger-mode`, the dtparam is called `sync-source` or `sync-sink`!

Will fix in next revision.